test-prow-e2e.sh

```diff
@@ -18,7 +18,7 @@ yarn build-libs
 yarn build-samples
 
 # Create a virtual X11 display via Xvfb for use with Cypress E2E testing
-Xvfb :99 -screen 0 1920x1080x24 &
+Xvfb :99 -screen 0 1920x1080x24 2>&1 > /dev/null &
```

Comment:
This redirects both stdout and stderr to `/dev/null` to avoid log pollution with non-actionable warnings like
```
The XKEYBOARD keymap compiler (xkbcomp) reports:
> Warning:          Could not resolve keysym XF86EmojiPicker
Errors from xkbcomp are not fatal to the X server
```